app/assets/javascripts/application.js

```diff
@@ -1,5 +1,68 @@
+/*
+  The following comments are parsed by Gulp include
+  (https://www.npmjs.com/package/gulp-include) which uses
+  Sprockets-style (https://github.com/sstephenson/sprockets)
+  directives to concatenate multiple Javascript files into one.
+*/
 //= include ../../../node_modules/govuk_frontend_toolkit/javascripts/vendor/polyfills/bind.js
 //= include ../../../bower_components/jquery/dist/jquery.js
+//= include ../../../bower_components/hogan/web/builds/3.0.2/hogan-3.0.2.js
 //= include ../../../bower_components/digitalmarketplace-frontend-toolkit/toolkit/javascripts/option-select.js
+//= include ../../../bower_components/digitalmarketplace-frontend-toolkit/toolkit/javascripts/list-entry.js
+//= include ../../../bower_components/digitalmarketplace-frontend-toolkit/toolkit/javascripts/word-counter.js
+//= include ../../../bower_components/digitalmarketplace-frontend-toolkit/toolkit/javascripts/validation.js
+//= include ../../../node_modules/govuk_frontend_toolkit/javascripts/govuk/selection-buttons.js
```

Comment:
We're also gong to need to add another file to call the selection-buttons code as this just adds their module to the `GOVUK` namespace.

The supplier app is a good example of how to do this and has a file for it we can use:

The file: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/assets/javascripts/_selection-buttons.js

Where it's added in application.js: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/assets/javascripts/application.js#L16